docs/conf.py

```diff
@@ -61,6 +61,8 @@
     ("py:class", "LightningDataModule"),
     # Undocumented class
     ("py:class", "torchvision.models.resnet.ResNet"),
+    ("py:class", "segmentation_models_pytorch.base.model.SegmentationModel"),
+    ("py:class", "segmentation_models_pytorch.unet.model.Unet"),
```

Comment:
Unet is documented: https://segmentation-modelspytorch.readthedocs.io/en/latest/docs/api.html#unet

Might need to change `__module__` though. SegmentationModel isn't documented as far as I can tell.